browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -110,7 +110,7 @@
 #endif
 
 #if BUILDFLAG(ENABLE_GLIC)
-#include "chrome/browser/glic/border_view.h"
+#include "chrome/browser/glic/glic_border_view.h"
```

Comment:
why is this needed here, don't we disable glic?

Reply:
cc: @simonhong

Reply:
if we don't set `enable_glic = false` we probably should be?

Reply:
Done in https://github.com/brave/brave-core/pull/27522/commits/cfca9ebbdc93782086015e7996ab4a779b985733